test/functional/change_stream.test.js

```diff
@@ -1975,6 +2145,7 @@ describe('Change Streams', function () {
         this.changeStream.on('resumeTokenChanged', resumeToken => {
           this.resumeTokenChangedEvents.push({ resumeToken });
         });
+        this.changeStream.isEmitter = false;
```

Comment:
This looks no longer relevant right? If we do the assigning of the symbol property in the constructor like I suggested then you can use getSymbolFrom in test/tools/utils.js to set this if we need it.